### PR TITLE
re-submission: One click install by default

### PR DESCRIPTION
**Pull request by martijnhoekstra**
Re-submission of the safe-keeping branch in #1130, submitted rebased but otherwise as-is.

See also, https://github.com/scala/scala-lang/pull/1130#issuecomment-644031929

> Following all the discussions and a suggestion of @julienrf, I have removed the changes to the download page, and only kept the blog post, so that we can move on by announcing the existence of the one-click install, encourage some people to test it more widely, but keep from recommending it by default until it's been tested more.
> 
> I have kept the changes to the download page in a separate branch for safekeeping: https://github.com/sjrd/scala-lang/tree/one-click-install-by-default

### CODE REVIEW

**Comment by sjrd** [comment]
@martijnhoekstra Thanks for resubmitting this!

Perhaps keep only the first commit in the PR, rather than having the third commit cancel the second one?

**Comment by martijnhoekstra** [comment]
> @martijnhoekstra Thanks for resubmitting this!
> 
> Perhaps keep only the first commit in the PR, rather than having the third commit cancel the second one?

I didn't notice that was the content of a separate commit, thanks for the heads-up!

**Comment by SethTisue** [comment]
I was on vacation last week, but I'll take a look this week.

File: _layouts/downloadpage.html [hunk 1]
```diff
@@ -16,23 +16,24 @@
...
         <div class="install-steps">
           <div class="step">
             <img src="/resources/img/download/arrow-asset.png" alt="">
             <div class="number-step">1</div>
             <div class="text-step">
-              <h3>First, make sure you have the Java 8 JDK (or Java 11 JDK) installed.</h3>
-              <p>To check, open the terminal and type:</p>
-              <p><code>java -version</code><span>(Make sure you have version 1.8 or 11.)</span></p>
-              <p><i>(If you don't have it installed, download Java from <a href="https://www.oracle.com/java/technologies/javase-jdk8-downloads.html">Oracle Java 8</a>, <a href="https://www.oracle.com/java/technologies/javase-jdk11-downloads.html">Oracle Java 11</a>, or <a href="https://adoptopenjdk.net/">AdoptOpenJDK 8/11</a>. Refer <a href="https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html">JDK Compatibility</a> for Scala/Java compatiblity detail.</i></p>
+              <h3>First, install Java, Scala, and all related command-line tools.</h3>
```
Comment (SethTisue) [inline _layouts/downloadpage.html:25]: This doesn't convey clearly that Coursier is going to handle all of that for me.

Suggested replacement: "Install Coursier, which will install everything else for you: Java, Scala, and related command-line tools."

File: _layouts/downloadpage.html [hunk 2]
```diff
@@ -42,34 +43,36 @@ <h3>Then, install Scala:</h3>
...
           </div>
 
           <p class="bottom-lead">
-            Compared to other programming languages, installing Scala is a bit unusual. Scala is unusual because it is usually installed for each of your Scala projects rather than being installed system-wide. Both of the above options manage (via sbt) a specific Scala version per Scala project you create.
-            <br><br>
-            But it's also possible to "install" Scala in numerous other ways; e.g., grab Scala binaries and use Scala from the command line or use Scala in your browser!
-            <br><br>
+            Compared to other programming languages, installing Scala is a bit unusual.
+            In addition to the system-wide installation mentioned above, build tools such as sbt will automatically manage a specific Scala version per Scala project you create.
+            This makes sure that there are no version conflicts between different Scala projects on your computer.
```
Comment (SethTisue) [inline _layouts/downloadpage.html:75]: > This makes sure that there are no version conflicts between different Scala projects on your computer

Perhaps: "This allows each Scala project on your computer to use any version of Scala it wants, without interfering with each other."

**Review by SethTisue — APPROVED** [review]
LGTM. I suggested a few possible wording changes.

**Comment by martijnhoekstra** [comment]
closing for now per scala/docs.scala-lang#1675

**Pull request closed without merging**